src/components/TextField/TextField.scss

```diff
@@ -337,6 +337,11 @@ $stacking-order: (
 .newDesignLanguage.TextField {
   color: var(--p-text);
 
+  .Backdrop {
+    border: 1px solid var(--p-border-neutral-subdued);
+    border-top: border(dark);
+  }
+
```

Comment:
I can't update the `border()` mixin as this is not a change we want across multiple components. `border(dark)` is used in a few places. I also hoped I could use `border(base)` for the subdued color but the contrast was not correct for this implementation.

Reply:
border-top leaves a hard edge. I wonder if using box-shadow give it a bit more of a natural look

![image](https://screenshot.click/All_Components__Text_field_-_All_Examples__Storybook_2020-08-25_13-45-22.png)

![image](https://screenshot.click/All_Components__Text_field_-_All_Examples__Storybook_2020-08-25_13-45-04.png)

Reply:
Oh that is nice, I will give that a go and test across browsers. Thanks @kyledurand!